### PR TITLE
Feat/client solana support

### PR DESCRIPTION
**Pull request by ericHgorski**

### CODE REVIEW

File: packages/client/src/client.ts [hunk 1]
```diff
@@ -118,6 +118,10 @@ export class SkipClient {
     this.getEVMSigner = options.getEVMSigner;
     this.getSVMSigner = options.getSVMSigner;
 
+    if (!Object.keys(this.endpointOptions).length) {
+      console.warn('Warning: You are using unreliable public endpoints. We strongly reccomend overriding them via endpointOptions for use beyond development settings.');
+    }
+
```
Comment (codingki) [inline packages/client/src/client.ts:124]: We shouldn't do this here, I think it's better to have the warning in `getRpcEndpointForChain`. if they not providing any rpc endpoint we output the warning.

**Review by codingki — CHANGES_REQUESTED** [review]

File: packages/client/src/client.ts (hunk 1 above)
Comment (codingki) [reply packages/client/src/client.ts:124]: https://github.com/skip-mev/skip-go/blob/950e8ee3865ad21fb60e68381be170e8d4bbe9eb/packages/client/src/client.ts#L1487

Comment (codingki) [reply packages/client/src/client.ts:124]: we can add it in that line

Comment (ericHgorski) [reply packages/client/src/client.ts]: fixed!

**Review by codingki — APPROVED** [review]
please do `npx changeset` and commit push before merging this.

**Pull request merged**